libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -873,7 +873,7 @@ class AP_AHRS {
 
     // return an airspeed estimate if available. return true
     // if we have an estimate
-    bool _airspeed_estimate(float &airspeed_ret, AirspeedEstimateType &status) const;
+    bool _EAS(float &airspeed_ret, AirspeedEstimateType &status) const;
```

Comment:
I'd prefer _airspeed_EAS and _airspeed_TAS

Reply:
Changed